Sources/FCM/Helpers/FCM+SendMessage.swift

```diff
@@ -34,7 +34,6 @@ extension FCM {
 
             return self.client.post(URI(string: url), headers: headers) { (req) in
                 struct Payload: Content {
-                    let validate_only: Bool = false
```

Comment:
Isn't it required?

Reply:
Didn't seem to be for me when I tested it. I'll have a look at the documentation.

Reply:
It doesn't explicitly say it [here](https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages/send) that is optional but the description does somewhat imply it:

> Flag for testing the request without actually delivering the message.

It seems to work fine without the property in my tests. Removing it was the easiest way to get rid of an Xcode warning that was related to the strict being `Decodable` (due to `Content`) while the property has a default value.